src/writer/WriterInterface.py

```diff
@@ -24,6 +24,8 @@ class WriterInterface(Module):
                               "attributes that can be used here. Type: list."
        "output_file_prefix", "The prefix of the file that should be created. Type: string."
        "output_key", "The key which should be used for storing the output in a merged file. Type: string."
+       "transparent_background", "If true, the background will be set to transparent and the alpha channel will be "
```

Comment:
This does not affect if the background is transparent or not, right? This only includes the alpha channel in the file, right?

Reply:
Yes that's correct, sorry I copied and pasted from the RGBrenderer file, shouldn't have done that

Reply:
No worries, that is the whole reason we do reviews ;)

Reply:
I guess it would be better to say:
> "transparent_background", "If true, the alpha channel will be written to file. Type: bool. Default: False."
Or do you think any more info is necessary?

Reply:
No, I don't think anything else is needed. Do you agree?

Reply:
No, I think that should be sufficient, I'll update the pull request. Thanks for your patience!

Reply:
Thanks for your contribution. Adding more features to BlenderProc will help a lot of users!